spring-cloud-netflix-eureka-server/src/test/java/org/springframework/cloud/netflix/eureka/server/InstanceRegistryTests.java

```diff
@@ -106,15 +106,7 @@ void testInternalCancel() {
 		// calling tested method
 		instanceRegistry.internalCancel(APP_NAME, HOST_NAME, false);
 		// event of proper type is registered
-		assertThat(this.testEvents.applicationEvents.size()).isEqualTo(1);
```

Comment:
The idea is not to remove the assertions. Please keep the assersions, but add necessary stubbing so that `cancelled`/ `renewed` return `true`.

Reply:
I agree. I changed this to registering the instance directly: It is too closely related to the internal implementation to utilize the stub object.